### ISSUE TITLE
[COMMENT including '\n' ]Skipping table integration: unable to generate ALTER TABLE due to use of unsupported features

### ISSUE THREAD

**Issue by jianhaiqing** [skeema/skeema#48]
TABLE definition : 
```
 CREATE TABLE `class_detail` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `uid` varchar(45) NOT NULL,
  `day` datetime NOT NULL,
  `stage` int(11) NOT NULL COMMENT '10种情况\n',
  `team` int(11) NOT NULL,
  `student` int(11) NOT NULL,
  `class_amount` int(11) NOT NULL,
  `no_team_class` int(11) NOT NULL,
  `is_deleted` tinyint(1) NOT NULL DEFAULT '0',
  `created_at` datetime NOT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=utf8
```

**Comment by evanelias** [comment]
Thank you for the bug report! Verified as described. Same bug also exists with \r.

Hopefully should be an easy fix. I'll investigate and post more details soon.

**Comment by evanelias** [comment]
I've just released [v1.1.2](https://github.com/skeema/skeema/releases/tag/v1.1.2), which contains a fix for this bug. Thanks again for the report!
